io/net/ethtool_test.py

```diff
@@ -201,6 +201,15 @@ def test_ethtool(self):
                                       ignore_status=True)
                     if ret.exit_status != 0:
                         self.fail("%s %s" % (self.args, ret.stderr_text))
+                    # Set queue values to original values
+                    cmd = "ethtool %s %s %s %s %s %s %s %s %s %s" % (
+                        self.args, self.interface, self.param[0], self.default_set[0],
+                        self.param[1], self.default_set[1], self.param[2],
+                        self.default_set[2], self.param[3], self.default_set[3])
```

Comment:
@vaishnavibhat can you re-frame this as below,

```
cmd = f"ethtool {self.args} {self.interface} {self.param[0]} {self.default_set[0]}
             { self.param[1]} {self.default_set[1]} {self.param[2]} {self.default_set[2]} {self.param[3]}    
             {self.default_set[3])}"
```

This will help, not to miss any parameter and reference to the variables.